website/static/js/koHelpers.js

```diff
@@ -436,6 +436,48 @@ ko.bindingHandlers.datePicker = {
 };
 
  /**
+ * Bind content of contenteditable to observable. Looks for maxlength attr
+ * and underMaxLength binding to limit input.
+ * Example:
+ * <div contenteditable="true" data-bind="editableHTML: <observable_name>" maxlength="500"></div>
+ */
+ko.bindingHandlers.editableHTML = {
+    init: function(element, valueAccessor, allBindings, bindingContext) {
+        var $element = $(element);
+        var initialValue = ko.utils.unwrapObservable(valueAccessor());
+        $element.html(initialValue);
+        $element.on('change input paste keyup blur', function() {
+            var observable = valueAccessor();
+            observable($element.html());
+        });
+    },
+    update: function(element, valueAccessor, allBindings, bindingContext) {
+        var $element = $(element);
+        var initialValue = ko.utils.unwrapObservable(valueAccessor());
+        var bindings = bindingContext;
+        var charLimit = $element.attr('maxlength');
+        if (charLimit && bindingContext.underMaxLength) {
+            var inputTextLength = $element[0].innerText.length;
+            var errorExceedsCharacterLimit = 'Exceeds character limit. Please reduce to ' + charLimit + ' characters or less.';
+            if (inputTextLength > parseInt(charLimit) + 1) {
```

Comment:
I'm not entirely sure but it looks like innerText counts a `<br>` as one character. I would think it is treated like a newline character since `\r\n` counts as two.

![screen shot 2016-06-14 at 9 10 40 am](https://cloud.githubusercontent.com/assets/7131985/16044017/9d96f114-3210-11e6-878c-2f9c3395cfbb.png)